blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandlerIntegrationTest.java

```diff
@@ -106,33 +129,19 @@ public void testMultiplotQueryWithEnum() throws Exception {
         JsonParser jsonParser = new JsonParser();
         JsonObject responseObject = jsonParser.parse(responseContent).getAsJsonObject();
 
-        String expectedResponse = String.format("{\n" +
-                "  \"metrics\": [\n" +
-                "    {\n" +
-                "      \"unit\": \"unknown\",\n" +
-                "      \"metric\": \"%s\",\n" +
-                "      \"data\": [\n" +
-                "        {\n" +
-                "          \"timestamp\": 1439231324001,\n" +
-                "          \"enum_values\": {\n" +
-                "            \"OK\": 1\n" +
-                "          }\n" +
-                "        },\n" +
-                "        {\n" +
-                "          \"timestamp\": 1439231324003,\n" +
-                "          \"enum_values\": {\n" +
-                "            \"ERROR\": 1\n" +
-                "          }\n" +
-                "        }\n" +
-                "      ],\n" +
-                "      \"type\": \"enum\"\n" +
-                "    }\n" +
-                "  ]\n" +
-                "}", metric_name);
-
-        JsonObject expectedObject = jsonParser.parse(expectedResponse).getAsJsonObject();
-
-        Assert.assertEquals(expectedObject, responseObject);
-        EntityUtils.consume(query_response.getEntity());
```

Comment:
We are reading the body, so its not required.  You only need to call ```EntityUtils.consume``` if you haven't read the response body.